_build/bootstrap.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+###
+#
+#  Script to bootstrap a node within docker.
+#  Example use:
+#
+#  exec docker run --detach --rm --name "akash-1" \
+#    -e AKASH_MONIKER=akash-1 \
+#    -e "AKASH_HOME=$AKASH_HOME" \
+#    -p "$LOCAL_IP:26657:26657" \
+#    -p 0.0.0.0:26656:26656 \
+#    -v "$STORAGE:$AKASH_HOME/data" \
+#    --mount type=bind,src=/root/bootstrap.sh,target=/bootstrap.sh \
+#    "$IMAGE" \
+#    /bootstrap.sh
+#
+###
+
+
+apt-get update
+apt-get install jq curl -y
+
+DEFAULT_META_URL=https://raw.githubusercontent.com/ovrclk/net/master/mainnet/meta.json
+META=./meta.json
+
+meta_url="${1:-$DEFAULT_META_URL}"
+
+curl -sSfl "$meta_url" > "$META"
+
+export AKASH_HOME="${AKASH_HOME:-/akash}"
```

Comment:
i think default dir is `.akash`

Reply:
but I hate that :joy:

Reply:
used `--home` because you never know :roll_eyes: